plugins/permission-node/api-report.md

```diff
@@ -14,6 +14,7 @@ import { DefinitivePolicyDecision } from '@backstage/plugin-permission-common';
 import { EvaluatorRequestOptions } from '@backstage/plugin-permission-common';
 import express from 'express';
 import { IdentifiedPermissionMessage } from '@backstage/plugin-permission-common';
+import { JsonSchema7Type } from 'zod-to-json-schema/src/parseDef';
```

Comment:
Not sure about this import tbh. An alternative might be `ReturnType<typeof zodToJsonSchema>`

Reply:
Think this is prolly better. Is the schema type omportant at all though? Could it be an opaque `object`?